ale_linters/tex/lacheck.vim

```diff
@@ -0,0 +1,51 @@
+" Author: Andrew Balmos - <andrew@balmos.org>
+" Description: lacheck for LaTeX files
+
+let g:ale_tex_lacheck_executable =
+\   get(g:, 'ale_tex_lacheck_executable', 'lacheck')
+
+function! ale_linters#tex#lacheck#Handle(buffer, lines) abort
+  " Mattes lines like:
+  "
+  " "book.tex", line 37: possible unwanted space at "{"
+  " "book.tex", line 38: missing `\ ' after "etc."
+
+  " stdin-wrapper makes files like: /tmp/tmp.0hV8ww029I/file.tex
+  let l:pattern = '^".\+/file.tex", line \(\d\+\): \(.\+\)$'
```

Comment:
Could you use a bit more of a liberal regex here in terms of the filename? At some point in the future, I will make the basename of the files match the original filenames, so when I finally get that in, it will break this code here with this pattern.